offchain/danaswap-api/test/Api.purs

```diff
@@ -0,0 +1,199 @@
+module Test.Api
+  ( openPoolWrongTokenRightRedeemer
+  , openPoolWrongTokenWrongRedeemer
+  , openPoolMultipleTokens
+  , depositLiquidityWrongTokenWrongRedeemer
+  , depositLiquidityWrongTokenRightRedeemer
+  ) where
+
+import Contract.Prelude
+
+import Contract.Address (scriptHashAddress)
+import Contract.Monad (Contract, liftContractM)
+import Contract.PlutusData (Datum(..), PlutusData(..), Redeemer(..), toData)
+import Contract.Prim.ByteArray (hexToByteArray)
+import Contract.ScriptLookups as Lookups
+import Contract.Scripts (mintingPolicyHash, validatorHash)
+import Contract.TxConstraints (DatumPresence(..))
+import Contract.TxConstraints as Constraints
+import Contract.Value (mkTokenName, mpsSymbol)
+import Contract.Value as Value
+import Ctl.Util (buildBalanceSignAndSubmitTx, getUtxos, waitForTx)
+import DanaSwap.Api (PoolId, Protocol, getPoolById)
+import DanaSwap.CborTyped (configAddressValidator)
+import Data.BigInt as BigInt
+import Data.Map as Map
+
+-- TODO rework this when the real open pool gets updated
+openPoolWrongTokenWrongRedeemer :: Protocol -> Contract () PoolId
+openPoolWrongTokenWrongRedeemer { poolVal, liquidityMP, poolIdMP, configUtxo } = do
+  poolID <- liftContractM "failed to make token name" $ mkTokenName =<< hexToByteArray "aaaa"
+  wrongID <- liftContractM "failed to make token name" $ mkTokenName =<< hexToByteArray "aabb"
+  let poolIdMph = mintingPolicyHash poolIdMP
+  poolIdCs <- liftContractM "hash was bad hex string" $ mpsSymbol poolIdMph
+  --liquidityMPH <- liftContractM "failed to hash mp" $ mintingPolicyHash liq
```

Comment:
Do we need this or will this be commented in, once the function has been reworked?

Reply:
I think I just forgot to delete it thanks for catching that.